src/App.jsx

```diff
@@ -14,33 +16,78 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getSorted(obj, way) {
+  const currentGoods = [...obj];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  currentGoods.sort((el1, el2) => {
+    switch (way) {
+      case 'is-info':
+        return el1.localeCompare(el2);
+      case 'is-success':
+        return el1.length - el2.length;
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return currentGoods;
+}
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [selectItem, setSelectItem] = useState('');
+  const [selReverse, setselReverse] = useState('');
+  const goods = getSorted(goodsFromServer, selectItem);
+
+  function reset() {
+    setselReverse('');
+    setSelectItem('');
+  }
+
+  if (selReverse) {
+    goods.reverse();
+  }
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={`button is-info ${selectItem === 'is-info' ? null : 'is-light'}`}
```

Comment:
This is a code quality issue. You are using ternary operator here to add 'is-light' class conditionally. However, when `selectItem === 'is-info'` is `true`, you are adding `null` to the class string. It would be better to use a logical AND operator (&&) to conditionally add 'is-light' class. Same applies to lines 63 and 71.